src/lib.rs

```diff
@@ -37,13 +47,27 @@ mod convert;
 
 #[cfg(any(
     all(any(target_arch = "x86", target_arch = "x86_64"), target_feature = "aes", not(miri)),
-    all(any(target_arch = "arm", target_arch = "aarch64"), target_feature = "crypto", not(miri), feature = "stdsimd")
+    all(
+        any(target_arch = "arm", target_arch = "aarch64"),
+        target_feature = "crypto",
+        not(miri),
+        feature = "stdsimd"
+    )
 ))]
 mod aes_hash;
 mod fallback_hash;
 #[cfg(test)]
 mod hash_quality_test;
 
+#[cfg(feature = "std")]
+/// [Hasher]: std::hash::Hasher
+/// [HashMap]: std::collections::HashMap
+/// Type alias for [HashMap]<K, V, std::hash::BuildHasherDefault<AHasher>>
+pub type HashMap<K, V> = std::collections::HashMap<K, V, std::hash::BuildHasherDefault<AHasher>>;
+#[cfg(feature = "std")]
+/// Type alias for [HashSet]<K, std::hash::BuildHasherDefault<AHasher>>
+pub type HashSet<K> = std::collections::HashSet<K, std::hash::BuildHasherDefault<AHasher>>;
```

Comment:
I think this should be using `ahash::RandomState` instead of `std::hash::BuildHasherDefault<AHasher>` here and above

Reply:
Yep you are right!